binrz/rz-test/run.c

```diff
@@ -89,9 +89,9 @@ static char *convert_win_cmds(const char *cmds) {
 static RzSubprocessOutput *run_rz_test(RzTestRunConfig *config, ut64 timeout_ms, const char *cmds, RzList *files, RzList *extra_args, bool load_plugins, RzTestCmdRunner runner, void *user) {
 	RzPVector args;
 	rz_pvector_init (&args, NULL);
-	rz_pvector_push (&args, "-escr.utf8=0");
-	rz_pvector_push (&args, "-escr.color=0");
-	rz_pvector_push (&args, "-escr.interactive=0");
+	rz_pvector_push (&args, "-e scr.utf8=0");
```

Comment:
Why? I think this is wrong, because by doing this there are actually two arguments ("-e" and "scr.utf8=0"). I think there should be no problem in keeping these lines as they were before. @XVilka